### DISCUSSION TITLE
How do I use this project without Composer?

### DISCUSSION THREAD

**Discussion by hiphigi** [thephpleague/commonmark#659] [Q&A] [answered]
This may be a stupid question, but how do you install it without composer? Which files do I need to include to have the namespaces and class available to me?

**Comment by GrahamCampbell** [comment, marked as answer]
Just google for how to autoload any PSR-4 project. The beauty of PSR-4 is write one autoloader, and it'll work anywhere.

**Comment by ghost** [comment]
Since it's possible, we can also read on the [main page of the League's website](http://thephpleague.com/):

> We distribute code via Packagist and Composer.

**Comment by colinodell** [comment]
I second @GrahamCampbell's advice.  While it's technically possible to download the ZIP and use it without an autoloader, it's going to be a major hassle, so I'd strongly advise against it.
